src/main/java/org/apache/cassandra/sidecar/CassandraSidecarDaemon.java

```diff
@@ -38,19 +38,22 @@
     private static final Logger logger = LoggerFactory.getLogger(CassandraSidecarDaemon.class);
     private final HttpServer server;
     private final Configuration config;
+    private final CDCReaderService cdcReaderService;
 
     @Inject
-    public CassandraSidecarDaemon(HttpServer server, Configuration config)
+    public CassandraSidecarDaemon(HttpServer server, Configuration config, CDCReaderService cdcReaderService)
     {
         this.server = server;
         this.config = config;
+        this.cdcReaderService = cdcReaderService;
     }
 
     public void start()
     {
         banner(System.out);
         validate();
         logger.info("Starting Cassandra Sidecar on {}:{}", config.getHost(), config.getPort());
+        cdcReaderService.start();
```

Comment:
Besides , do you think we should add a flag to enable or disable the cdc reader service?

Reply:
Good suggestion. Let me add that, so users who don't need this can just keep it disabled.

Reply:
yes